geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/sortedset/AbstractZIncrByIntegrationTest.java

```diff
@@ -168,7 +168,7 @@ public void shouldCreateNonExistentKeyWithInfiniteScore_whenIncrementIsNegativeI
 
```

Comment:
I know these tests weren't changed in this PR, but you could reuse that `assertKeyIsCreatedWithIncrementOf` method in the `memberShouldBeCreatedWhenItDoesNotExist_withIncrementOfNegative/PostiveInfinity` tests.